sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 ## Release History
 
-### 3.12.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+### 3.12.0 (2021-09-24)
+#### New Features
+* Added support for UUID id type to spring data cosmos SDK.
```

Comment:
Please mention the azure-cosmos version update.

Reply:
Like this -> Updated `azure-cosmos` to version 4.19.1.

Reply:
done